client/analysisui/layoutactionmanager.js

```diff
@@ -5,12 +5,28 @@ var backbone = require('backbone');
 var LayoutAction = require('./layoutaction');
 var SuperClass = require('../common/superclass');
 
-var LayoutActionManager = function(actions) {
+var LayoutActionManager = function(view) {
 
-    this._actionsSource = actions;
+    this._view = view;
     this._actions = [];
     this._resources = { };
     this._executingActions = 0;
+    this._initialisingData = 0;
+
+    this.beginInitialisingData = function() {
```

Comment:
would you like me to just go through all the code and change and initialising to initializing because this is not the only place it has been used.